figures/views.py

```diff
@@ -392,6 +393,125 @@ def get_serializer_context(self):
         return context
 
 
+class CourseMonthlyMetricsViewSet(CommonAuthMixin, viewsets.ViewSet):
+    """
+
+    """
+    months_back = 6
```

Comment:
Should we store this value in an editable setting?

Reply:
@melvinsoft upcoming, this will be modifiable via query param, but yes, good idea to have a default in the settings, thanks!